docs/src/test-api/class-testconfig.md

```diff
@@ -239,7 +239,7 @@ export default defineConfig({
 
 Metadata contains key-value pairs to be included in the report. For example, HTML report will display it as key-value pairs, and JSON report will include metadata serialized as json.
 
-See also [`property: TestConfig.populateGitInfo`] that populates metadata.
+Providing `'git.commit.info': {}` property will populate it with the git commit details. This is useful for CI/CD environments.
```

Comment:
add the property to the example below to make it clear that the value should be just an empty object